launch_testing/test/check_env.py

```diff
@@ -0,0 +1,4 @@
+if __name__ == '__main__':
+    import os
+    assert(os.environ.get('testenv2', 'Not Set') == 'Not Set')
```

Comment:
Assert is not a function but a keyword. This will always be true since the arguments are a tuple.